docs/user/rstudio/ide/guide/tools/copilot.qmd

```diff
@@ -0,0 +1,237 @@
+
+For general advice on how to use Copilot, please see:\
+- [How to use GitHub Copilot: Prompts, tips, and use cases](https://github.blog/2023-06-20-how-to-write-better-prompts-for-github-copilot/)\
+- [How to use GitHub Copilot](https://docs.github.com/en/copilot/getting-started-with-github-copilot/how-to-use-github-copilot)\
+- [Learning GitHub Copilot video series](https://learn.microsoft.com/en-us/shows/introduction-to-github-copilot/)
+
+::: {.callout-warning title="Trust but Verify"}
+While Copilot often generates useful and functional code, it is important to note that the suggestions are not always valid code examples or completely solve the intended problem. Copilot suggestions are non-deterministic and Posit does not guarantee the quality, accuracy, or security of the outputs. It is important to review the suggestions and ensure that they are both accurate and appropriate for the intended use case. Copilot may generate code that contains insecure coding patterns, bugs, or outdated practices. You should always use best practices in code review, testing, and adherence to security standards when using Copilot. For more information, see the [FAQs on GitHub Copilot](https://github.com/features/copilot#faq) and [GitHub Copilot for Business](https://docs.github.com/en/copilot/overview-of-github-copilot/about-github-copilot-for-business).
+:::
+
+### Autocomplete vs Copilot
+
+The existing autocomplete in RStudio is a static list of possible completions based on the currently focused characters. Copilot is a dynamic predicted suggestion that is generated based on the current context, including comments, code elsewhere in the document or project, and the currently focused characters. Copilot will also provide suggestions for code that is not currently in the active document.
+
+Similar to normal autocomplete, Copilot suggestions will begin after a short delay. If you want to prompt Copilot directly in a new file, you can start typing a descriptive variable or function name and then pause for a moment. Copilot will then generate a suggestion based on the current context.
+
+At times, normal autocomplete and Copilot may seem to conflict with each other. In these cases, it is best to review the Copilot suggestion and determine if it is appropriate for the current context. If it is, you can accept the suggestion by pressing <kbd>Tab</kbd>. If it is not, you can ignore the suggestion and continue typing or force the normal autocomplete to show by pressing <kbd>Ctrl+Space</kbd>.
+
+There is also a setting in the Tools \> Global Options \> Copilot options to make <kbd>Tab</kbd> prefer "Copilot Suggestion" or "Autocomplete Suggestion". This setting defaults to prefer "Copilot Suggestion" if Copilot is enabled.
```

Comment:
We don't have this option. Unless I'm missing it?

Reply:
We used to: https://github.com/rstudio/rstudio/pull/13521

But I think that it was removed per: https://github.com/rstudio/rstudio/pull/13521#issuecomment-1699507163

I'll remove the "setting" paragraph.

Reply:
removed